DEPRECTED/blockchainLevelRepository.js

```diff
@@ -1,3 +1,6 @@
+/**
+ * DEPRECTAED
```

Comment:
Every file this is misspelled. It is DEPRECATED. Also the folder name is DEPRECTED instead of DEPRECATED